proposed/2020/Transitive-Dependencies.md

```diff
@@ -75,6 +75,37 @@ When a user highlights a transitive package, they will see a pop-up that display
 
 Finally, selecting a transitive dependency will provide the user the ability to “Install” the package as you normally would a top-level dependency. This will promote the transitive dependency to a top-level dependency, and the transitive dependency will now be placed in the top-level packages list instead.
 
+#### Detailed design
+
+**Collapsible vertical tabs:** Direct and transitive packages will be grouped under collapsible tabs 
+
+* **Labels:** Tabs will be labeled “Top-level packages” and “Transitive packages” respectively 
+* **Count:** Tabs will display the number of packages in the group 
+* **Default state:** Transitive packages tab will be collapsed by default 
+* **Warnings:** Tab headers will surface a warning icon for vulnerabilities and deprecations - see “Vulnerabilities and deprecation” section for relevant details 
+
+**Package items:** The packages under the “Transitive packages” tab will have the same appearance/design as the “Top level packages” (Id, description, author, versions on the right, etc.) 
+* **Status overlays:** Transitive packages will not show an “Update available” icon (upward arrow) or “Latest version installed” icon (check mark) - only directly installed packages will 
+* **Tooltip:** Hovering over the package list item will show a pop-up that displays how the transitive dependency originated & what top-level package(s) are bringing it in. 
+
+**Search:** Searching for a package ID will filter the packages shown in both tabs in the same way. 
+* Searching for “Microsoft” will show all relevant filtered results for both the top level and transitive packages 
+* If one tab or the other is collapsed, they will remain collapsed on search 
+* The counter for the number of packages will change to reflect the number of packages being shown when filtered with a search query 
+
+**Package details window:** The package details window will be the exact same for transitive and top level packages. 
+* Similar to top-level packages, users will have the ability to “Install” or “Update” the transitive package to make it a top-level package of the desired version. 
+* PackageReference & Package.Config: PackageReference will be the priority for this feature – Packages.Config won’t be considered unless it comes for free with the work for PR. 
+
+**Browse, Installed, and Updates tab behavior:**
+* **Installed:** Transitive dependencies will only be shown and managed in the Installed tab 
+* **Browse:** We will not surface transitive dependencies in the Browse tab. Transitively installed packages will not have an “Installed latest version” check mark. 
+* **Updates:** We will not show transitive dependencies in the Updates tab. This is because users shouldn’t typically update transitive dependencies outside of extenuating circumstances like vulnerabilities and deprecation. 
```

Comment:
``This is because users shouldn’t typically update transitive dependencies outside of extenuating circumstances like vulnerabilities and deprecation. ``

And if they did, it would be by "Installing" the transitive dependency as a top-level package to manually manage it.